components/event-bus/.gitignore

```diff
@@ -11,3 +11,4 @@ bin/
 docker/
 vendor/
 /unit-tests.xml
+cmd/event-publish-service/event-publish-service
```

Comment:
Maybe add a comment and do the same for all binaries? Otherwise this feels highly specific.